Lambda-School-Labs/CS10-quizzer
# Brandon/stripe unsubscribe

# Description

Users are now able to unsubscribe from a payment plan

Fixes # (issue)

## Type of change

Please delete options that are not relevant.


# How Has This Been Tested?
- Create teacher
- Signup for either one of the payment plans
- Check database for a teachers `CustomerID`
- Click the `Cancel Subscription` button in the billing section
- Check database for a lack of a `CustomerID` on that teacher

# Checklist:
